paseos/communication/find_next_window.py

```diff
@@ -0,0 +1,59 @@
+import pykep as pk
+from loguru import logger
+
+from .get_communication_window import get_communication_window
+
+
+def find_next_window(
+    local_actor,
+    link_name: str,
+    target_actor,
+    search_window_in_s: float,
+    t0: pk.epoch,
+    search_step_size: float = 10,
+):
+    """Returns the start time of the next window in the given timespan (if any).
+
+    Args:
+        local_actor (BaseActor): Actor to find window from.
+        link_name (str): Name of the comm device.
+        target_actor (BaseActor): Actor find window with.
+        search_window_in_s (float): Size of the search window in s.
+        t0 (pk.epoch): Start time of the search.
+        search_step_size (float): Size of steps in the search. Defaults to 10.
+
+    Returns:
+        Window start (pk.epoch), window length (float [s]), data (int [b]). None,0,0 if none found.
+    """
+    logger.debug(f"Find next comms window between {local_actor} and {target_actor}")
+
+    assert link_name in local_actor.communication_devices, (
+        "Trying to use a not-existing communication link with the name: "
+        + local_actor.communication_devices
+    )
+    local_actor_comm_link = local_actor.communication_devices[link_name]
+
+    assert local_actor_comm_link.bandwidth_in_kbps > 0, "Bandiwidth has to be positive."
+    assert search_step_size > 0, "dt has to be positive."
+
+    # Compute start and end of the search window
+    t = t0.mjd2000 * pk.DAY2SEC
```

Comment:
make sure t0 is not None as in get_communication_window?

Reply:
Hm, then we should be checking all variables for correct types, would avoid to minimize overhead?